### PR TITLE
Functions bot

### PR DESCRIPTION
**Pull request by alancast**
Changing to a functions bot if it works

### CODE REVIEW

File: messages/run.csx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+// Bot Storage: Register the optional private state storage for your bot. 
+
+// (Default) For Azure Table Storage, set the following environment variables in your bot app:
+// -UseTableStorageForConversationState set to 'true'
+// -AzureWebJobsStorage set to your table connection string
+
+// For CosmosDb, set the following environment variables in your bot app:
+// -UseCosmosDbForConversationState set to 'true'
+// -CosmosDbEndpoint set to your cosmos db endpoint
+// -CosmosDbKey set to your cosmos db key
```
Comment (remyjette) [inline messages/run.csx]: No need for these comments, this bot should never require private state storage (or for that matter, hold any state at all between requests)

File: messages/run.csx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                    //         await client.Conversations.ReplyToActivityAsync(reply);
+                    //     }
+                    // }
+                    // break;
+                case ActivityTypes.ContactRelationUpdate:
+                case ActivityTypes.Typing:
+                case ActivityTypes.DeleteUserData:
+                case ActivityTypes.Ping:
+                default:
+                    log.Error($"Unknown activity type ignored: {activity.GetActivityType()}");
```
Comment (remyjette) [inline messages/run.csx]: We expect to get these activity types all the time in a Teams bot, no need to log an error

File: messages/run.csx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+                    //         reply.Text = "Welcome";
+                    //         if (!string.IsNullOrEmpty(newMember.Name))
+                    //         {
+                    //             reply.Text += $" {newMember.Name}";
+                    //         }
+                    //         reply.Text += "!";
+                    //         await client.Conversations.ReplyToActivityAsync(reply);
+                    //     }
+                    // }
+                    // break;
```
Comment (remyjette) [inline messages/run.csx]: Commented code makes me sad :(

**Review by remyjette — APPROVED** [review]
wow this is much cleaner :)

File: messages/run.csx (hunk 3 above)
Comment (alancast) [reply messages/run.csx]: removed

File: messages/run.csx (hunk 2 above)
Comment (alancast) [reply messages/run.csx]: removed

File: messages/run.csx (hunk 1 above)
Comment (alancast) [reply messages/run.csx]: removed

**Pull request merged**